api/v1/menus/getAll.js

```diff
@@ -1,4 +1,9 @@
+const DBClient = require('../../lib/DBClient');
+const dbInstance = new DBClient();
+
 module.exports = (req, res) => {
-    // TODO: getAll menu method.
-    return res.send('TODO: getAll menu method.');
-};
+    dbInstance.get('menus')
+        .then((resp) => res.status(200).json(resp))
```

Comment:
Please take a look at #14 and let me know if you think it's better to use this module instead of defining status codes directly.